video/bot/__init__.py

```
@@ -0,0 +1,2 @@
+from . import main
```

Ruff (0.8.2)

`.main` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)